### ISSUE TITLE
Gremlin support for Linux

### ISSUE THREAD

**Issue by BenjaBobs** [Azure/azure-cosmos-db-emulator-docker#88]
#37 Was just closed and though the reason wasn't communicated, I expect it was because strictly speaking it originally just tracked Linux support in general, which was achieved.

Then there is #59 which tracks gremlin support in general.
I worry that it is also going to be closed without further communication since technically there is gremlin support on windows.

So I made this issue to unambiguously request very specifically gremlin support for linux.

Gremlin support for linux would not only speed our CI pipeline up, but also significantly improve our developer experience, as our team has multiple members running linux on their dev machines.

**Comment by schamal2205** [comment]
i guess that this will never be supported...

**Comment by BenjaBobs** [comment]
Yeah, we've abandoned CosmosDb in favor of Postgres so we don't have to deal with these kinds of issues.

**Comment by martinfletchersustainware** [comment]
I raised an issue within the .NET Aspire repository relating to Gremlin support - This is a really big productivity boost, especially with the push of .NET Aspire!

I 100% agree, support for Gremlin is urgently needed to make the local CosmosDB development experience refined and suitable for larger enterprises and apps.

Here is the issue that is being tracked in the .NET Aspire repository - Hopefully these can align soon and support for Gremlin possible :)

https://github.com/dotnet/aspire/issues/10804

**Comment by martinfletchersustainware** [comment]
@sajeetharan - Any update on this at all? Either way would be fine, its just very hard to plan when no updates are provided.

From the activity on this repository, it does seem like this is maybe becoming a dead project?

**Comment by sajeetharan** [comment]
@martinfletchersustainware Thanks for the suggestion. We don’t currently have plans to formally support Gremlin on Linux. Our priorities are NoSQL and MongoDB/DocumentDB support.

**Comment by martinfletchersustainware** [comment]
Can I ask why no Gremlin support? Is it because is it not used enough or that it is a part of Cosmos that has no future?
